src/main/scala/scalafix/sbt/ScalafixPlugin.scala

```diff
@@ -233,13 +230,8 @@ object ScalafixPlugin extends AutoPlugin {
       .withConfig(jutil.Optional.ofNullable(config.map(_.toPath).orNull))
       .withPaths(paths.asJava)
 
-    if (paths.nonEmpty || shellArgs.explicitlyListsFiles) {
-
-      if (paths.lengthCompare(1) > 0) {
-        logger.info(
-          s"Running scalafix on ${paths.size} Scala sources"
-        )
-      }
+    if (paths.nonEmpty) {
+      logger.info(s"Running scalafix on ${paths.size} Scala sources")
```

Comment:
This should only print when fixing more than one file
```suggestion
    if (paths.lengthCompare(1) > 0) {
      logger.info(s"Running scalafix on ${paths.size} Scala sources")
    }
```